.eslintrc.yml

```diff
@@ -0,0 +1,11 @@
+env:
+  browser: true
+  es6: true
+extends: 'eslint:recommended'
+globals:
+  Atomics: readonly
+  SharedArrayBuffer: readonly
+parserOptions:
+  ecmaVersion: 2018
+  sourceType: module
+rules: {}
```

Comment:
Let's delete this file and use the correct linter config instead. See previous comments for the appropriate links.